Sw 641 add range slider to axiom

clairebw opened the pull request:
This adds a Range slider with two handles.
![Screen Shot 2019-04-08 at 15 47 50](https://user-images.githubusercontent.com/18306439/55729142-dbcd7900-5a15-11e9-82dd-50fb4f271719.png)

lpoulter commented on packages/axiom-components/src/Slider/Range.js:
```diff
@@ -0,0 +1,228 @@
+        values[1] = values[0];
+        break;
+      default:
+        return;
```
Might be nice to throw/log an error here.

lpoulter commented on packages/axiom-components/src/Slider/Range.js:
```diff
@@ -0,0 +1,228 @@
+    const { values } = this.state;
+
+    switch (event.key) {
+    case 'ArrowLeft':
```
Formatting looks strange here should `case` be indented?

clairebw replied:
The eslint rule explicitly requires that there is no indentation for `switch` cases: https://github.com/clairebw/axiom-react/blob/04ef2645fbce630b80043fd6539262c27ddf348a/packages/eslint-config-axiom/src/eslintrc.json#L18

lpoulter commented:
Hi @clairebw thanks for this. Looks like there are some lint and test issues on the build. Are you able to loop in design and link the new component in the docs once it build?

clairebw commented:
@lpoulter @TomWija @MariamBashir Thanks for your reviews. I fixed the tests, addressed the comments and @lilybrandwatch also approved it. Would you mind having a second look?
